### ISSUE TITLE
PHP SDK - Incorrect monolog package version requirements

### ISSUE THREAD

**Issue by john-peterson-g17** [configcat/php-sdk#32]
### Describe the bug

When attempting to install the v7.0.0 of the php-sdk I run into dependency conflicts. It seems that no matter what I do I can not get the package to install due to the strict monolog version requirement that conflicts with other packages. 

### To reproduce

Attempt to install the `configcat/configcat-client` on a project using php8.1 and Laravel 9.7

### Recommended Fix

Adjust the monolog package version requirement in the composer.json for the configcat/php-sdk repository.

```json
{
// Current
  "require": {
    "monolog/monolog": "2.0|^3.0", // <-- Currently Allows  =2.0 and >=3.0
  },

// Change to
  "require": {
    "monolog/monolog": "^2.0|^3.0", // <-- Allow  >=2.0 and >=3.0
  }
}
```

### Expected behavior

The `configcat/configcat-client` package to install successfully

#### Output of `composer require configcat/configcat-client` command

```
Using version ^7.0 for configcat/configcat-client
./composer.json has been updated
Running composer update configcat/configcat-client --with-all-dependencies
Loading composer repositories with package information
Cannot create cache directory /var/www/.composer/cache/repo/https---gitlab.com-api-v4-group-52288415---packages-composer/, or directory is not writable. Proceeding 
without cache
Cannot create cache directory /var/www/.composer/cache/repo/https---repo.packagist.org/, or directory is not writable. Proceeding without cache
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - monolog/monolog[2.0.0-beta2, ..., 2.0.2] require php ^7.2 -> your php version (8.1.7) does not satisfy that requirement.
    - spatie/laravel-ignition[1.0.0, ..., v1.x-dev] require monolog/monolog ^2.3 -> satisfiable by monolog/monolog[2.3.0, ..., 2.x-dev].
    - You can only install one version of a package, so only one of these can be installed: monolog/monolog[2.0.0-beta1, ..., 2.x-dev, 3.0.0-RC1, ..., 3.x-dev].    
    - configcat/configcat-client v7.0.0 requires monolog/monolog 2.0|^3.0 -> satisfiable by monolog/monolog[2.0.0, 3.0.0-RC1, ..., 3.x-dev].
    - Root composer.json requires configcat/configcat-client ^7.0 -> satisfiable by configcat/configcat-client[v7.0.0].
    - Root composer.json requires spatie/laravel-ignition ^1.0 -> satisfiable by spatie/laravel-ignition[1.0.0, ..., v1.x-dev].

You can also try re-running composer require with an explicit version constraint, e.g. "composer require configcat/configcat-client:*" to figure out if any version 
is installable, or "composer require configcat/configcat-client:^2.1" if you know which you need.

Installation failed, reverting ./composer.json to its original content.
```

#### composer.json in my project

```json
{
    "require": {
        "php": "^8.1",
        "aptive/pestroutes-sdk": "^4.7",
        "aws/aws-sdk-php": "^3.192",
        "crazybooot/base64-validation": "1.*",
        "docusign/esign-client": "^6.10",
        "dompdf/dompdf": "^2.0",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.5",
        "laravel/framework": "^9.7",
        "laravel/sanctum": "^2.15",
        "laravel/tinker": "^2.7",
        "league/flysystem-aws-s3-v3": "^3.0",
        "maatwebsite/excel": "^3.1",
        "okta/jwt-verifier": "^1.4",
        "predis/predis": "^2.0",
        "sendgrid/sendgrid": "^7.9",
        "tymon/jwt-auth": "^1.0"
    },
    "require-dev": {
        "barryvdh/laravel-ide-helper": "^2.10",
        "brianium/paratest": "^6.6",
        "spatie/laravel-ignition": "^1.0",
        "fakerphp/faker": "^1.9.1",
        "laravel/sail": "^1.0.1",
        "mockery/mockery": "^1.4.2",
        "nunomaduro/collision": "^6.1",
        "nunomaduro/larastan": "^2.1",
        "pcov/clobber": "^2.0",
        "phpunit/phpunit": "^9.3.3",
        "squizlabs/php_codesniffer": "^3.6"
    }
}
```

### SDK version

v7.0.0

### Language/Framework version

PHP v8.1, Laravel v9.7

### Platform

Docker PHP:8.1-fpm Image

**Comment by z4kn4fein** [comment]
Hi @PetersonG17, thanks for reporting this issue!
I've released `v7.0.1` with the suggested solution. Let us know if you have further issues.

**Comment by john-peterson-g17** [comment]
@z4kn4fein Thank you! :)

**Comment by john-peterson-g17** [comment]
I can verify that this fix in v7.0.1 resolves the issue. Thank you! 😄
